debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/pointer/helper/J9ROMClassHelper.java

```diff
@@ -125,7 +125,7 @@ public static boolean finalizeNeeded(J9ROMClassPointer romclass) throws CorruptD
 		return romclass. extraModifiers().allBitsIn(J9AccClassFinalizeNeeded);
 	}
 
-	public static boolean isClonable(J9ROMClassPointer romclass) throws CorruptDataException {
+	public static boolean isCloneable(J9ROMClassPointer romclass) throws CorruptDataException {
```

Comment:
This looks like an API of some sort...

Reply:
@keithc-ca

Reply:
It may be API - I'm not sure, but I couldn't find any references (neither in code nor via google).